test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
To avoid a lot of noise in the PR, I'm prob going to make `args` optional with a little magic (e.g. `arguments.length` sniffing or `args[args.length - 1]` sniffing)